dolphin-dnn/src/test/java/edu/snu/dolphin/dnn/NeuralNetworkTest.java

```diff
@@ -119,8 +137,8 @@ public void buildNeuralNetwork() throws InjectionException {
   @Test
   public void feedForwardTest() {
     final INDArray[] activations = neuralNetwork.feedForward(input);
-    assertTrue(Nd4jUtils.equals(activations, expectedActivations, tolerance));
     assertTrue(Nd4jUtils.equals(activations[activations.length - 1], expectedOutput, tolerance));
+    assertTrue(Nd4jUtils.equals(activations, expectedActivations, tolerance));
```

Comment:
An array of activations includes the output of a neural network. If the array of activations that the neural network generates is equal to `expectedActivations` within `tolerance`, the next line before change, that checks the equality between the last element of `activations` and `expectedOutput` was totally unnecessary.
However, if the order is reversed, we may easily figure out the cause of test failure cases such as one of activation values is not returned despite the correct output value.